components/omega/doc/devGuide/Driver.md

```diff
@@ -0,0 +1,58 @@
+(omega-dev-driver)=
+
+## Driver
+
+The Omega Driver consists of methods to run the Omega ocean model in either
+standalone mode or as a component of E3SM. In both cases, the execution of Omega
+is divided into three phases, carried out by the `ocnInit`, `ocnRun`, and
+`ocnFinalize` methods. An interface layer to translate data types of the parent
+model to and from internal Omega data types will also be needed for running
+Omega as a component. For standalone mode, a `main` function is defined to
+serve as the top-level driver. The driver methods will evolve as more modules
+are developed and as Omega is integrated with E3SM.
+
+### ocnInit
+
+The `ocnInit` method initializes everything needed to run the Omega ocean model,
+and has the following interace:
+```c++
+int ocnInit(MPI_Comm Comm, Calendar &OmegaCal, TimeInstant &StartTime, Alarm &EndAlarm);
```

Comment:
`ocnInit` returns a calendar, but neither `ocnRun` nor `ocnFinalize` use it. I am assuming this is done for some future use ?

Reply:
Yeah, the Calendar may potentially be needed for output, some forcings, or other capabilities added later. Also, the TimeInstant and the Alarm (via its TimeInstant members) are dependent on a Calendar, containing pointers to the Calendar they're based in, so I keep it in scope by declaring the Calendar in `main`